cmd/wfx/cmd/root/southbound.go

```diff
@@ -42,20 +44,32 @@ func createSouthboundCollection(schemes []string, storage persistence.Storage) (
 	}
 
 	swaggerJSON, _ := restapi.SwaggerJSON.MarshalJSON()
-	mw := middleware.NewGlobalMiddleware(restapi.ConfigureAPI(api),
-		[]middleware.IntermediateMW{
-			// LIFO
-			logging.MW{},
-			jq.MW{},
-			fsMW,
-			swagger.NewSpecMiddleware(api.Context().BasePath(), swaggerJSON),
-			health.NewHealthMiddleware(storage),
-			version.MW{},
-			middleware.PromoteWrapper(cors.AllowAll().Handler),
-		})
 
+	var pluginMws []middleware.IntermediateMW
+	if pluginsDir != "" {
+		pluginMws, err = createPluginMiddlewares(pluginsDir, quitChan)
+		if err != nil {
+			return nil, fault.Wrap(err)
+		}
+	}
+
+	// LIFO, i.e. middlewares are applied in reverse order
```

Comment:
I.e., lexicographic order on the filenames, right?

Reply:
No, this comment applies to the local variable `intermdiateMws` which is an array of middlewares (not just plugins but also filters (jq)).